### DISCUSSION TITLE
Can't change color of box shadow and can't change title of unordered list items

### DISCUSSION THREAD

**Discussion by AminulIslam254** [Atri-Labs/atrilabs-engine#607] [Help: Miscellaneous]
There are two problem's that I am facing currently
1. I can't change the color for the box shadow
2. I can't change the title and description for the items in unordered list
If anyone could answer these It would be really helpful

**Comment by ramchaudhary2000** [comment]
box-shadow: rgba(143, 144, 166, 32%) 0px 5px 17px, rgb(29, 174, 255) 0px 0px 0px 1px;

change the color by your requirement

**Reply by AminulIslam254** [reply]
> box-shadow: rgba(143, 144, 166, 32%) 0px 5px 17px, rgb(29, 174, 255) 0px 0px 0px 1px;
> 
> change the color by your requirement

Still did't work the atri framework isn't selecting the color whether I am typing it in the RGB section or manually selecting it by placing the white circle.Can you please tell me whether you are able to select the color for box shadow?
